spec/support/json_schemas.rb

```diff
@@ -17,12 +16,17 @@ def load_json_schema(file_name)
 
   def load_yaml_schema(file_name)
     File.open("#{Rails.root}/swagger/v1/#{file_name}") do |file|
-      YAML.safe_load(file.read)
+      begin
+        YAML.safe_load(file.read)
+      rescue Psych::SyntaxError => e
+        # Include original filename in exception to make debugging a less cryptic affair
+        raise Psych::SyntaxError.new(file_name, e.line, e.column, e.offset, e.problem, e.context)
```

Comment:
nice 👍

Reply:
This bugged me enough to fix it, spent too long scratching my head trying to figure out which file was causing issues.